metricbeat/docs/fields.asciidoc

```diff
@@ -1075,87 +1072,87 @@ type: long
 
 format: bytes
 
-Data size of couch documents associated with a node (bytes)
+Data size of Couch docs associated with a node (bytes).
 
 
 [float]
 === couchbase.node.couch.spatial.data_size.bytes
 
 type: long
 
-couchSpatialDataSize field
+Size of object data for spatial views (bytes).
```

Comment:
Reviewers: The description seemed to be missing here. Please verify that this is an accurate description of this field.